src/View/ViewVarsTrait.php

```diff
@@ -98,4 +98,30 @@ public function set(array|string $name, mixed $value = null)
 
         return $this;
     }
+
+    /**
+     * Wrapper function to set variables into the template as well as enabling the serialization of these variables
+     *
+     * @param array|string $name A string or an array of data.
+     * @param mixed $value Value in case $name is a string (which then works as the key).
+     *   Unused if $name is an associative array, otherwise serves as the values to $name's keys.
+     * @return $this
+     */
+    public function setSerialized(array|string $name, mixed $value = null)
```

Comment:
I'm concerned that this is only supported by `SerializedView`. The doc summary is misleading since it doesn't enable the serialization of the variables, only sets  `serialize` option.

I'm not sure of a better approach other than making SerializedView support an option that includes the variable data instead of just an array of variable names.

Reply:
Is this a method we could have on viewBuilder instead? Usage would be something like `$this->viewBuilder()->setSerialized(['cat', 'dog']);`

Reply:
My main concern is just to reduce 2 separate method calls with basically the same arguments to one.

Reply:
markstory's suggestion is adding the same method you propose, just on `ViewBuilder` instead of `Controller`.

Reply:
If it's in ViewBuilder, maybe we could just call it `serialize()`.